Packs/PopularCybersecurityNews/Integrations/PopularNews/PopularNews.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+import requests
+from bs4 import BeautifulSoup
+from CommonServerPython import *  # noqa: F401
+
+TABLE = []
+VERIFY = demisto.params()['insecure']
+
+
+def scrape_kos():
```

Comment:
Hi @teizenman. I will need to update this piece of code as part of maintaining the functionality of this pack. Could you please take a look when you have the time ? The instance I initially contributed this Pack from is now gone so I figured instead of creating a new PR, its best to check with you to see if you can help with updating the pack. I have already run the snippet below through the validation and linting checks so it should be fine.

Basically change this scrape_kos() function to the code below:
```
def scrape_kos():
    articles = []
    links = []
    dates = []
    response = requests.get("https://krebsonsecurity.com/", verify=VERIFY)
    soup = BeautifulSoup(response.text, "html.parser")

    for article in soup.select(".entry-title"):
        title = article.get_text().strip()
        articles.append(title)
        link = article.find('a').attrs['href']
        links.append(link)

    for date in soup.select(".adt"):
        dates.append(date.find('span').get_text().strip())

    return list(zip(articles, list(zip(links, dates))))

``` 

Thank you!

The release note/change notes is basically:
`Fix to better handle news from Krebs on Security.`